x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_status_tags.tsx

```diff
@@ -20,54 +27,97 @@ const BadgeItem = styled.div`
 
 const TagContainer = styled.div`
   padding: 10px;
-  max-height: 200px;
+  max-height: 229px;
   overflow: hidden;
 `;
 
+const OtherDiv = styled.div`
+  padding-left: 18px;
+`;
+
 interface Props {
   locations: MonitorLocation[];
 }
 
+interface StatusTag {
+  label: string;
+  timestamp: number;
+}
+
 export const LocationStatusTags = ({ locations }: Props) => {
   const {
     colors: { gray, danger },
   } = useContext(UptimeSettingsContext);
 
-  const upLocs: string[] = [];
-  const downLocs: string[] = [];
+  const upLocations: StatusTag[] = [];
+  const downLocations: StatusTag[] = [];
 
   locations.forEach((item: any) => {
     if (item.summary.down === 0) {
-      upLocs.push(item.geo.name);
+      upLocations.push({ label: item.geo.name, timestamp: new Date(item.timestamp).valueOf() });
     } else {
-      downLocs.push(item.geo.name);
+      downLocations.push({ label: item.geo.name, timestamp: new Date(item.timestamp).valueOf() });
     }
   });
 
+  // Sort by recent timestamp
+  upLocations.sort((a, b) => {
+    return a.timestamp < b.timestamp ? 1 : b.timestamp < a.timestamp ? -1 : 0;
+  });
+
+  moment.locale('en', {
+    relativeTime: {
+      future: 'in %s',
+      past: '%s ago',
+      s: 'few sec',
+      ss: '%s sec',
```

Comment:
Something like this yields 
<img width="272" alt="image" src="https://user-images.githubusercontent.com/18429259/72178924-fc19d200-33b1-11ea-9496-7a6af7ed397a.png">

Which I think is acceptable, and should represent the vast majority of use cases.

Here's one for the last minute:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/18429259/72178952-0cca4800-33b2-11ea-879a-6b9d3dbe7457.png">